src/main/scala/nl.knaw.dans.easy.deposit/Submitter.scala

```diff
@@ -110,7 +114,9 @@ class Submitter(stagingBaseDir: File,
     _ <- setRightsRecursively(draftDepositDir)
     // EASY-1464 step 3.3.9 Move copy to submit-to area
     _ = logger.info(s"moving $draftDepositDir to $submitDir")
-    _ = srcProvider.move(draftDepositDir.path, submitDir.path, StandardCopyOption.ATOMIC_MOVE)
+    _ <- Try(srcProvider.move(draftDepositDir.path, submitDir.path, StandardCopyOption.ATOMIC_MOVE)).recoverWith {
+      case e: FileAlreadyExistsException => Failure(AlreadySubmittedException(id))
```

Comment:
this may also cause an [`AtomicMoveNotSupportedException`](https://docs.oracle.com/javase/8/docs/api/java/nio/file/AtomicMoveNotSupportedException.html) also, although that is very unlikely...

Reply:
which is covered by https://github.com/DANS-KNAW/easy-deposit-api/blob/33cfadddd01f9be86dcd4b475ed6a4ef46c47044/src/main/scala/nl.knaw.dans.easy.deposit/Errors.scala#L49-L51